include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -1752,6 +1765,29 @@ namespace internal
                       task_info.cell_partition_data.back() * n_lanes);
     }
 
+    // set up map: deal.II index -> MatrixFree index
+    {
+      const auto &tria     = dof_handler[0]->get_triangulation();
+      const auto  mg_level = this->get_mg_level();
+
+      mf_cell_indices.resize((mg_level == numbers::invalid_unsigned_int) ?
+                               tria.n_active_cells() :
+                               tria.n_cells(mg_level),
+                             numbers::invalid_unsigned_int);
+
+      for (unsigned int cell = 0;
+           cell < n_cell_batches() + n_ghost_cell_batches();
+           ++cell)
+        for (unsigned int v = 0; v < n_active_entries_per_cell_batch(cell); ++v)
+          {
+            const auto tria_cell = get_cell_iterator(cell, v);
+            mf_cell_indices[(mg_level == numbers::invalid_unsigned_int) ?
+                              tria_cell->active_cell_index() :
+                              tria_cell->index()] =
+              cell * VectorizedArrayType::size() + v;
+          }
```

Comment:
This won't compile, you need to be in a member function of `MatrixFree` to be able to use these variables. (See also the CI failures.)

Reply:
@kronbichler I have the code back to the old place. Let's us the information later...